src/clientManager.js

```diff
@@ -135,4 +135,3 @@ ClientManager.prototype.autoload = function(/* sockets */) {
 		});
 	}, 1000);
 };
-
```

Comment:
Was that intentional? We should always keep an EOF line to keep git and other tools happy :-)